### ISSUE TITLE
Change Training Data Generation script

### ISSUE THREAD

**Issue by santacruz91** [CSE547-kollerz-s9wischu/CSE547-ColdStart#24]
We need a set of empty columns for the training set, but where there are values in validation / test set.
